client/lib/url/support.js

```diff
@@ -1,6 +1,7 @@
 const root = 'https://support.wordpress.com';
 
 export default {
+	ACCOUNT_RECOVERY: `${ root }/account-recovery`,
 	ADDING_GOOGLE_APPS_TO_YOUR_SITE: `${root}/adding-g-suite-to-your-site`,
```

Comment:
I tidied up in https://github.com/Automattic/wp-calypso/pull/8955/commits/3a40091503fdf4b7b778b1e513b9d808d1e9653d